packages/lesswrong/themes/userThemes/darkMode.ts

```diff
@@ -270,6 +270,7 @@ export const darkModeTheme: UserThemeSpecification = {
       votedCandidate: "#8A5751",
       ctaBackground: "#64342E",
       ctaText: "#FFF",
+      rhsLink: '#E7714E',
```

Comment:
@agnesstenlund For the RHS links, I thought the red was too dark in dark mode, so I made them this orange that matches the right side of the header gradient. Feel free to change this color :)